java/src/main/native/ai_onnxruntime_OrtSession.c

```diff
@@ -23,6 +23,7 @@ JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtSession_createSession__JJLjava_la
     const jchar* cPath = (*jniEnv)->GetStringChars(jniEnv, modelPath, NULL);
     size_t stringLength = (*jniEnv)->GetStringLength(jniEnv, modelPath);
     wchar_t* newString = (wchar_t*)calloc(stringLength+1,sizeof(jchar));
```

Comment:
>jchar [](http://example.com/codeflow?start=64&length=5)

We need sizeof(wchar_t)

Reply:
I think it doesn't matter. jchar is "unsigned short", and wchar_t is 2 bytes on Windows.